sam-styles/pagination/_structure.scss

```diff
@@ -98,5 +98,65 @@ text-decoration: none;
 }
 
 .sam-pagination__expanded > li > a > img{
+  color: #123050;
+}
+
+.sam-pagination__expanded > a.active {
+	color: black;
+}
+
+.sam-pagination__expanded > a:hover{
+	text-decoration: none;
+}
+
+.sam-pagination__expanded > a > img{
   height: 1rem;
 }
+
+/*******************************
+        SB ADDED SORT
+  *******************************/
+
+.sam-sort__wrapper{
+	display: flex;
+	flex-direction: column;
+}
```

Comment:
Please move all code below this line inside `.sam-sort__wrapper`
`.sam-sort__wrapper {
    .sort-header{}
    .sort-content > option{}
    ...
}`